kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java

```diff
@@ -85,7 +85,7 @@ public KyuubiBeeLine(boolean isBeeLine) {
   @Override
   void usage() {
     super.usage();
-    output("Usage: java \" + KyuubiBeeLine.class.getCanonicalName()");
+    output("Usage: java " + KyuubiBeeLine.class.getCanonicalName());
```

Comment:
this is an independent issue, you can fix a in an independent PR first :)

Reply:
ok